lib/px/pxconfig.lua

```diff
@@ -60,6 +60,7 @@ _M.collector_host = string.format('collector-%s.perimeterx.net', _M.px_appId)
 _M.client_host = "client.perimeterx.net"
 _M.collector_port_overide = nil
 _M.client_port_overide = nil
+_M.reverse_xhr_disabled  = false
```

Comment:
why not something more consistent with `first_party_enabled `, like `first_party_xhr_enabled`?